applications/ShapeOptimizationApplication/python_scripts/response_functions/plane_packaging_response.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import print_function, absolute_import, division
+
+import time as timer
+
+import KratosMultiphysics
+from KratosMultiphysics import Logger
+
+from .packaging_response_base import PackagingResponseBase
+from .. import custom_math as cm
+
+import numpy as np
+
+class PlanePackagingResponse(PackagingResponseBase):
+
+    def __init__(self, identifier, response_settings, model):
+
+        super().__init__(identifier, response_settings, model)
+
+        self.plane_origin = []
+        for i in range(0,3):
+            self.plane_origin.append(self.response_settings["plane_origin"][i].GetDouble())
+
+        self.plane_normal = []
+        for i in range(0,3):
+            self.plane_normal.append(self.response_settings["plane_normal"][i].GetDouble())
+
+        self.plane_normal = cm.ScalarVectorProduct(1.0/cm.Norm2(self.plane_normal), self.plane_normal)
```

Comment:
Hmm, I don't quite like the heavy use of the custom_math capabilities here. Simply because we should get rid of of the whole script in favor of standard capabilities. But I understand its necessary here.

Reply:
I agree, but it is better to use it as long as it is there, instead of implementing dot product and similar again in this file.